apps/next-product-starter/src/styles/cueplayer.css

```diff
@@ -0,0 +1,158 @@
+
+
+/* cue note */
```

Comment:
@DrShpongle I brought this over so it would render but it has a couple of issues:

* it's all tailwind specific and needs to be scss in the /packages/player package 
* the play-progress-bar styles completely change the play progress bar so it can't inline like normal. this probably needs to be an optional/configurable approach so the bar can be "floating" or "inline"

Reply:
What if we extract `ProgressControl` out of `ControlBar` component and put it just under the video holder? I tried it, seems work well.